extensions/parquet/base/src/main/java/io/deephaven/parquet/base/materializers/ObjectMaterializerBase.java

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.parquet.base.materializers;
+
+import io.deephaven.parquet.base.PageMaterializer;
+
+import java.util.Arrays;
+
+// TODO Is it okay if I use this class as a base class for the other object type materializers, like LocalDate,
```

Comment:
It's OK. Decide if you want to keep strongly typed arrays (in which case just construct them inline in the `super` constructor call) or use `Object` arrays (and don't bother casting to `TYPE[]`.